lua/telescope/utils.lua

```diff
@@ -147,6 +148,83 @@ utils.path_tail = (function()
   end
 end)()
 
+utils.slice = function(tbl, s, e)
+  local pos, new = 1, {}
```

Comment:
Do you actually want to copy all elements between s and e into a new table? Thats not great.

Maybe this plenary module can help https://github.com/nvim-lua/plenary.nvim/blob/master/lua/plenary/iterators.lua
No idea how it works tho. But i saw a range function. This is the spec file https://github.com/nvim-lua/plenary.nvim/blob/master/tests/plenary/iterators_spec.lua